docker-compose.local.yaml

```
@@ -31,7 +34,29 @@ services:
     ports:
       - "8080:8080"
     tty: true
-  
+
+  #############################################################################################
+  ###                                       Database                                        ###
+  #############################################################################################
+  dpia-db:
```

I'm 100% ok if we just want to call this postgres tbh. If we bring in any other third party tools we don't have to annotate them with "dpia-"
```suggestion
  postgres:
```